tests/PPC/TestLifting.cpp

```diff
@@ -0,0 +1,800 @@
+      {{"pc", uint64_t(0x12)}, {"r5", uint64_t(0x1337)}, {"r6", uint64_t(0x0)}},
+      {{"pc", uint64_t(0x12 + 4)},
+       {"r5", uint64_t(0x1337)},
+       {"r6", uint64_t(0x5)}},
+      reg_to_accessor);
+
+#if LLVM_VERSION_NUMBER < LLVM_VERSION(15, 0)
+  curr_context.enableOpaquePointers();
+#endif
+  TestSpecRunner<PPCState> runner(curr_context, remill::kArchPPC);
+  runner.RunTestSpec(spec);
+}
+
+// Convert Floating-Point Double-Precision from Signed Integer
+TEST(PPCVLELifts, PPCVLEConvertDoubleFromSignedInteger) {
+  llvm::LLVMContext curr_context;
+  // efdcfsi r5, r4
+  std::string insn_data("\x10\xa0\x22\xf1", 4);
+  TestOutputSpec<PPCState> spec(
+      0x12, insn_data, remill::Instruction::Category::kCategoryNormal,
+      {{"pc", uint64_t(0x12)}, {"r4", uint64_t(0x1337)}, {"r5", uint64_t(0x0)}},
+      {{"pc", uint64_t(0x12 + 4)},
+       {"r4", uint64_t(0x1337)},
+       {"r5", uint64_t(0x40b3370000000000)}},
+      reg_to_accessor);
+
+#if LLVM_VERSION_NUMBER < LLVM_VERSION(15, 0)
+  curr_context.enableOpaquePointers();
+#endif
+  TestSpecRunner<PPCState> runner(curr_context, remill::kArchPPC);
+  runner.RunTestSpec(spec);
+}
+
+// Convert Floating-Point Single-Precision from Signed Integer
+TEST(PPCVLELifts, PPCVLEConvertFloatFromSignedInteger) {
+  llvm::LLVMContext curr_context;
+  // efscfsi r5, r4
+  std::string insn_data("\x10\xa0\x22\xd1", 4);
+  TestOutputSpec<PPCState> spec(
+      0x12, insn_data, remill::Instruction::Category::kCategoryNormal,
+      {{"pc", uint64_t(0x12)}, {"r4", uint64_t(0x1337)}, {"r5", uint64_t(0x0)}},
+      {{"pc", uint64_t(0x12 + 4)},
+       {"r4", uint64_t(0x1337)},
+       {"r5", uint64_t(0x4599b800)}},
+      reg_to_accessor);
+
+#if LLVM_VERSION_NUMBER < LLVM_VERSION(15, 0)
+  curr_context.enableOpaquePointers();
+#endif
+  TestSpecRunner<PPCState> runner(curr_context, remill::kArchPPC);
+  runner.RunTestSpec(spec);
+}
+
+// Convert Floating-Point Single-Precision to Signed Integer
+TEST(PPCVLELifts, PPCVLEConvertFloatToSignedInteger) {
+  llvm::LLVMContext curr_context;
+  // efsctsi r5, r4
+  std::string insn_data("\x10\xa0\x22\xd5", 4);
+  TestOutputSpec<PPCState> spec(0x12, insn_data,
+                                remill::Instruction::Category::kCategoryNormal,
+                                {{"pc", uint64_t(0x12)},
+                                 {"r4", uint64_t(0x4599b800)},
+                                 {"r5", uint64_t(0x0)}},
+                                {{"pc", uint64_t(0x12 + 4)},
+                                 {"r4", uint64_t(0x4599b800)},
+                                 {"r5", uint64_t(0x1337)}},
+                                reg_to_accessor);
+
+#if LLVM_VERSION_NUMBER < LLVM_VERSION(15, 0)
+  curr_context.enableOpaquePointers();
+#endif
+  TestSpecRunner<PPCState> runner(curr_context, remill::kArchPPC);
+  runner.RunTestSpec(spec);
+}
+
+// Test syscall
+// TODO(wtan): Disabled, callother not supported
+TEST(PPCVLELifts, DISABLED_PPCVLESyscall) {
```

Comment:
Is this captured on the PR or an issue we may want to start adding support for lifting these to hypercalls